utpsrv.cpp

```diff
@@ -0,0 +1,247 @@
+/*
+     This file is part of libhttpserver
+     Copyright (C) 2011, 2012, 2013, 2014, 2015 Sebastiano Merlino
+
+     This library is free software; you can redistribute it and/or
+     modify it under the terms of the GNU Lesser General Public
+     License as published by the Free Software Foundation; either
+     version 2.1 of the License, or (at your option) any later version.
+
+     This library is distributed in the hope that it will be useful,
+     but WITHOUT ANY WARRANTY; without even the implied warranty of
+     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+     Lesser General Public License for more details.
+
+     You should have received a copy of the GNU Lesser General Public
+     License along with this library; if not, write to the Free Software
+     Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+     USA
+*/
+
+#include <httpserver.hpp>
+#include <thread>
+#include <map>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+const int SMALL_CONSTANT = 2;
+
+
+class config_keeper
```

Comment:
It seems like you're not using this class anywhere or maybe you just haven't gotten to actually using it yet. I could be wring though